halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/Secret.java

```diff
@@ -11,4 +11,5 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Target(ElementType.FIELD)
 public @interface Secret {
+    boolean alwaysDecrypt() default false;
```

Comment:
Nit. Could we add a comment description for `alwaysDecrypt` and when it should be set to true?

Reply:
Good point. Added.